build_web_compilers/README.md

```diff
@@ -64,6 +64,34 @@ targets:
           - -O2
 ```
 
+### Configuring -D environment variables
```

Comment:
We probably should just support these similar to experiments.... 🤣.

Anyways I would list the command line usage as well because I think that would be the most common/useful?

Reply:
> We probably should just support these similar to experiments....

Would we invalidate the entire build when they change? We could potentially use the zone to know which builders read them...

Want to file a feature issue?

> Anyways I would list the command line usage as well because I think that would be the most common/useful?

I'm not really sure which is the common usage. The `--define` approach is ugly, I added an example below.